src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -3339,126 +3230,90 @@
-  "botania.page.tinyPotato2": "The potato can be given items to display, by right clicking them onto it, one for each side of the block.$(p)Items placed in the bottom face will show up in the lower half of the front.$(p)Potatoes are not exactly known for their dexterity, so some items may look out of place.",
-  "botania.page.tinyPotato1": "You are being believed in",
+  "botania.tagline.tinyPotato": "A friend for all of us",
+  "botania.page.tinyPotato0": "It's a tiny potato! It believes in you!$(p)No. Really, that's it. A $(1)Potato$(0), thrown into a $(1)Mana Pool$(0), gains a little spark of life and joy. The $(1)Tiny Potato$(0) is lively, but patting it with a right-click will make it even more so!$(p)A Potato can also be named in an $(1)Anvil$(0).",
+  "botania.page.tinyPotato2": "The potato can be given items (with right-click) to hold; one item per side of the Potato. (Items placed on its bottom face will show up on the lower half of the Potato's front.)$(p)Unfortunately, potatoes aren't exactly known for their dexterity, so some items may look out of place when held. (And that's okay.)",
+  "botania.page.tinyPotato1": "Believe in it, who believes in you",
 
   "botania.entry.headCreating": "Head Creation",
   "botania.tagline.headCreating": "Create any player's head",
-  "botania.page.headCreating0": "A typical incantation of the Elves, combining a few resources and a $(1)Name Tag$(0) allows for the creation of a replica of sorts of the head of the person described in the tag.$(p)How it works isn't really known, but it seems to be possible to replicate it with a $(1)Runic Altar$(0).",
-  "botania.page.headCreating2": "A subset of the community has dedicated themselves to creating heads that serve as all sorts of decorations. Unlike any other page in this lexicon, clicking the $(1)View Online$(0) button on the left of this page will take you to a website with a list of names that can be used for this purpose. Note, however, that this list is not affiliated with Botania in any way and is definitely not the only place to find these types of heads.",
+  "botania.page.headCreating0": "A commonly-invoked spell of the Elves is the use of a few resources and a $(1)Name Tag$(0) to transmute a piece of $(1)Livingrock$(0) into the head of the $(4)Player$(0) named by the tag.$(p)Its exact mechanics are unknown, but the spell seems easy enough to replicate on a $(1)Runic Altar$(0).",
+  "botania.page.headCreating2": "$(l:https://freshcoal.com/usernames.php)A subset of the $(4)Minecraft$(0) community (whoops fourth wall) has dedicated itself to creating heads that serve as all sorts of decorations. Clicking on this page's text will take you to a website with a list of names usable for this purpose. Do note, however, that this list is not affiliated with Botania in any way, shape, or form, and is definitely not the only place to find these types of heads.$(/l)",
```

Comment:
link is applying to the entire page
also could you change the link to https://freshcoal.com/maincollection? It's a more modern version of that old one.